### PR TITLE
Added my blog

### PR DESCRIPTION
**Pull request by saarthakchats**
There's Code-In Me --- My Blog

### CODE REVIEW

**Review by abishekvashok — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks, please see if you are able to provide a link to test the changes or screenshots depicting them. :smile:

**Comment by saarthakchats** [comment]
screenshot 55

It shows.

File: index.html [hunk 1]
```diff
@@ -1023,6 +1023,15 @@ <h5 class="uppercase mb0">My dive into open source with FOSSASIA during Google C
 		                    </a>
 		                </div>
 
+		                <div class="col-md-3 col-sm-6">
+							<a href="https://saarthakchaturvedi05.wordpress.com/2017/12/06/theres-code-in-me/" target="_blank">
```
Comment (anshumanv) [inline index.html]: Please indent your code properly, it's best to have well-indented code! :smile:

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -1023,6 +1023,15 @@ <h5 class="uppercase mb0">My dive into open source with FOSSASIA during Google C
...
+								<div class="image-tile outer-title text-center">
+									<img src="img/blog/Saarthak.jpg" height="240" width="240">
+									<div class="title mb16">
+										<h5 class="uppercase mb0">There's Code-In Me</h5>
+										<span>Saarthak Chaturvedi</span>
+									</div>
+
```
Comment (abishekvashok) [inline index.html]: Pls close the div

**Comment by anshumanv** [comment]
Thanks for the screenshot @saarthakchats 
Looks amazing! :smile:

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
Done Indenting

File: index.html [hunk 3]
```diff
@@ -1023,6 +1023,15 @@ <h5 class="uppercase mb0">My dive into open source with FOSSASIA during Google C
 		                    </a>
 		                </div>
 
+		                <div class="col-md-3 col-sm-6">
+				<a href="https://saarthakchaturvedi05.wordpress.com/2017/12/06/theres-code-in-me/" target="_blank">
+					<div class="image-tile outer-title text-center">
+						<img src="img/blog/Saarthak.jpg" height="240" width="240">
+						<div class="title mb16">
```
Comment (anshumanv) [inline index.html:1030]: This div is yet to be closed. Please fix

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
Changes made

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Awesome! :smile: 
Thanks for incorporating the changes quickly.

Please learn how to squash your commits and please squash commits from next time on. 

Cheers!

**Comment by abishekvashok** [comment]
@saarthakchats good job, thank you

**Pull request merged**